Gemfile

```diff
@@ -3,5 +3,5 @@ source 'https://rubygems.org'
 gemspec
 
 group :release do
-  gem 'github_changelog_generator',  :require => false, :git => 'https://github.com/github-changelog-generator/github-changelog-generator' if RUBY_VERSION >= '2.2.2'
```

Comment:
or https://github.com/voxpupuli/github-changelog-generator/tree/voxpupuli_essential_fixes ??

Reply:
Since this is so rarely released and it usually works on small modules I'd like to avoid it. Especially since this is not managed in any way.